conversation.go

```diff
@@ -1,7 +1,7 @@
 package slack
 
 // Conversation is the foundation for IM and BaseGroupConversation
-type conversation struct {
```

Comment:
This shouldn't be exported. The fields within conversation are exported and can be accessed when embedded.